config/routes.rb

```diff
@@ -1,13 +1,14 @@
 Rails.application.routes.draw do
   get "/", to: "items#index"
+  get '/cart', to: "cart#show"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.